pagination/src/main/java/com/cerner/beadledom/pagination/models/OffsetPaginatedListDto.java

```diff
@@ -0,0 +1,97 @@
+package com.cerner.beadledom.pagination.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.auto.value.AutoValue;
+import com.wordnik.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * Offset based pagination list.
+ *
+ * <p>Represents an offset based paginated list in a rest response.
+ *
+ * @author John Leacox
+ * @since 3.1
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@AutoValue
+@JsonPropertyOrder({
+    "items",
+    "totalResults",
+    "firstLink",
+    "lastLink",
+    "nextLink",
+    "prevLink"
+})
+@JsonDeserialize(builder = OffsetPaginatedListDto.Builder.class)
+public abstract class OffsetPaginatedListDto<T> {
```

Comment:
Looking over the dependencies, there aren't a huge number of service-specific ones that would concern me, especially since currently the client would have to do a concrete implementation of the service anyway, so they end up having similar dependencies.